examples/chip-tool/commands/common/CHIPCommand.h

```diff
@@ -96,7 +96,7 @@ class CHIPCommand : public Command
         AddArgument("trace_decode", 0, 1, &mTraceDecode);
 #endif // CHIP_CONFIG_TRANSPORT_TRACE_ENABLED
         AddArgument("trace-to", &mTraceTo, "Trace destinations, comma-separated (" SUPPORTED_COMMAND_LINE_TRACING_TARGETS ")");
-        AddArgument("ble-adapter", 0, UINT16_MAX, &mBleAdapterId);
+        AddArgument("ble-adapter", 0, UINT16_MAX, &mBleAdapterId, "Bluetooth adapter index. 0 -> hci0, 1 -> hci1, etc.");
```

Comment:
The exact names depend on the operating system and whatnot, right?

Reply:
Yes, I have pushed a new version which references the docs for details on the controller naming in particular platforms. I wrote such a description for Linux as a start.